docs/glossary.rst

```diff
@@ -53,29 +62,24 @@ Glossary
 
    lite-function
    lite-functions
-      A lite weight version of an existing `plasmapy` function,
-      typically a formulary function.  These functions are intended for
-      computational use where speed matters more than anything else, and
-      as such **all argument safeguards have been stripped away** to
-      leave the most fundamental calculation possible.  A few things
-      to know about lite-functions:
-
-      - Lite-functions are a flavor of existing `plasmapy`
-        functionality and are denoted with a trailing ``_lite`` to the
-        name. For example,
-        `~plasmapy.formulary.speeds.thermal_speed_lite` is the
-        lite-function associated with
-        `~plasmapy.formulary.speeds.thermal_speed`.
-      - Lite-functions are typically bound to their normal version as
-        the ``lite`` attribute.  For example,
-        `~plasmapy.formulary.speeds.thermal_speed_lite` can be
-        accessed as ``thermal_speed.lite``.
-      - Lite-functions view all input and output arguments as being in
-        SI units.
-      - **UNITS ARE NOT VALIDATED BY THE FUNCTIONS.**  It is solely up to
-        the user to pass in parameters with the correct units.
-      - When possible, lite-functions are decorated with `numba.jit`
-        (or the like) for an additional speed boost.
+      A lightweight version of an existing `plasmapy` function intended
+      for applications where computational efficiency is most important.
+      While most `~plasmapy.formulary` functions accept |Quantity|
+      objects created using `astropy.units`, lite-functions accept
+      numbers and |array_like| inputs that are implicitly assumed to be
```

Comment:
The `|array_like|` is not working correctly.  Not sure what the reason is at the moment.

Reply:
It's because your branch is not up-to-date with `main`, so `|array_like|` is not defined in `common_links.rst`.

Reply:
Just updated it — thanks!